shell/egl_window.cc

```diff
@@ -49,8 +49,9 @@ EglWindow::EglWindow(size_t index,
   FML_DLOG(INFO) << "+ EglWindow()";
 
   m_surface = wl_compositor_create_surface(m_display->GetCompositor());
-  assert(m_surface);
-  FML_DLOG(INFO) << "EglWindow::m_surface = " << static_cast<void*>(m_surface);
+#if HIDPI_CLIENT
```

Comment:
Why only for HDPI_CLIENT? 

What's the reasoning. There's no commit description explaining it.